data/transaction.go

```diff
@@ -116,11 +116,11 @@ type Event struct {
 
 // PreparedResults is the DTO that holds all the results after processing
 type PreparedResults struct {
-	Transactions    []*Transaction
-	ScResults       []*ScResult
-	Receipts        []*Receipt
-	AlteredAccounts map[string]*AlteredAccount
-	DeploysInfo     []*ScDeployInfo
+	Transactions []*Transaction
+	ScResults    []*ScResult
+	Receipts     []*Receipt
+	AlteredAccts *AlteredAccounts
```

Comment:
why using pointers? You increased the code coupling.

Reply:
refactored.